getting_started_scenarios/ecr_scenario/SPECIFICATION.md

```diff
@@ -0,0 +1,301 @@
+# Amazon Elastic Container Registry Service Scenario Specification
+
+## Overview
+This SDK getting started scenario demonstrates how to interact with Amazon Elastic Container Registry Service (Amazon ECR) using the AWS SDK. It demonstrates various tasks such as creating a repository, setting an ECR repository policy, obtaining an authorization token, pushing a docker image to the repository, and so on.  Finally this scenario demonstrates how to clean up resources. Its purpose is to demonstrate how to get up and running with Amazon ECR and the AWS SDK.
+
+## Resources and User Input
+The required resources for this SDK scenario are an IAM role and a local Docker image. The IAM role must have permission to interact with the Amazon ECR service. To create an IAM role, see [Creating IAM roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html). 
```

Comment:
"permission to interact with the Amazon ECR service" is a little vague. Can this be more specific?

Reply:
This is a topic that is discussed in the User Guide. I provided a cross ref where its all explained.

Reply:
I will put in the sample pol into the Readme.